src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp

```diff
@@ -1819,6 +1832,40 @@ class spell_halion_spawn_living_embers : public SpellScriptLoader
         }
 };
 
+class spell_halion_blazing_aura : public SpellScriptLoader
+{
+    public:
+        spell_halion_blazing_aura() : SpellScriptLoader("spell_halion_blazing_aura") { }
+
+        class spell_halion_blazing_aura_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_halion_blazing_aura_SpellScript);
+
+            void FilterTargets(std::list<WorldObject*>& targets)
+            {
+                targets.remove_if([this](WorldObject* target) { return target->GetEntry() != NPC_LIVING_EMBER; });
```

Comment:
issue adressed, it needs conditions for both difficulties of the spell, i was using only for the first ^^